CHANGES/4247.feature

```diff
@@ -0,0 +1 @@
+Automated running autobahn test suit by integrating with pytest.
```

Comment:
"suit" has a different meaning :)
```suggestion
Automated running autobahn test suite by integrating with pytest.
```

Reply:
Also, it's not a user-facing feature. It doesn't change anything for users, only for devs/maintainers. So it's probably ".misc".

Reply:
Make it `4247.1.misc` then. Towncrier understands multiple fragments per issue with an extra number like that.